src/sqlfluff/rules/L061.py

```diff
@@ -0,0 +1,64 @@
+"""Implementation of Rule L061."""
+
+from typing import Optional
+
+from sqlfluff.core.parser.segments.raw import CodeSegment
+from sqlfluff.core.rules.base import BaseRule, LintFix, LintResult, RuleContext
+from sqlfluff.core.rules.doc_decorators import document_fix_compatible
+from sqlfluff.core.rules.functional import sp
+
+
+@document_fix_compatible
+class Rule_L061(BaseRule):
+    """Use ``!=`` instead of ``<>`` for ``not equal to`` comparison.
```

Comment:
True, although I do think it should be quoted?

`Use ``!=`` instead of ``<>`` for "not equal to" comparisons.`

Otherwise it reads like it saying "if does not equal a comparison"

Reply:
Agreed. Updated suggested fix.